types/react-navigation-fluid-transitions/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for react-navigation-fluid-transitions 0.2
+// Project: https://github.com/fram-x/FluidTransitions#readme
+// Definitions by: My Self <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped\
+// TypeScript Version: 2.8
```

Comment:
This is old

Reply:
vscode is on 3.1.4

Reply:
I need this version because the library needs to be on the same version as react navigation

Reply:
The TypeScript version listed here should be the lowest version needed for the language features used here, so it makes sense for it to match the version on its dependency.